src/components/Calculator.js

```diff
@@ -0,0 +1,95 @@
+/* eslint-disable react/prefer-stateless-function */
+import React from 'react';
+import calculate from '../logic/calculate';
+
+export default class Calculator extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {};
+    this.handleClick = this.handleClick.bind(this);
+  }
+
+  handleClick(e) {
+    this.setState((state) => calculate(state, e.target.textContent));
+  }
+
+  render() {
+    const { next, total } = this.state;
+    return (
+      <div className="calc-container">
+        <div className="screen">
+          {next || total || 0 }
+        </div>
+        <div className="button-box">
+          <div className="button-row">
+            <button type="button" className="button-column" onClick={this.handleClick}>
+              AC
+            </button>
+            <button type="button" className="button-column" onClick={this.handleClick}>
+              +/-
+            </button>
+            <button type="button" className="button-column" onClick={this.handleClick}>
+              %
+            </button>
+            <button type="button" className="button-column orange" onClick={this.handleClick}>
+              *
+            </button>
```

Comment:
- [ ] (Required) This button should be the ` ÷` symbol, not the `*` symbol as you have it in your project. It would be nice to change it to the appropriate symbol so that your calculator can run as expected. Currently, when I click this symbol and press the `=` symbol, a blank screen is shown. 👍